src/clientsettingsdlgbase.ui

```diff
@@ -1151,6 +1201,9 @@
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
+                  <property name="buddy">
```

Comment:
Does name=buddy mean? No translation required or what does it stand for?

Reply:
```
$ grep -n edtAlias src/clientsettingsdlgbase.ui
92:                     <cstring>pedtAlias</cstring>
141:                   <widget class="QLineEdit" name="pedtAlias">
1344:  <tabstop>pedtAlias</tabstop>
```
The "buddy" is the field this field labels.  It's the _internal_ field name - i.e. what's used in the C++ code, not what's displayed.  All the label widgets _should_ have one.